packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_load_bundle_task.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+
+import 'method_channel_firestore.dart';
+
+import 'utils/exception.dart';
+
+class MethodChannelLoadBundleTask extends LoadBundleTaskPlatform {
+  MethodChannelLoadBundleTask(this._task, this._bundle, this._firestore)
```

Comment:
Is that constructor made to be used by other people?

Reply:
I believe so, every other constructor [here](https://github.com/FirebaseExtended/flutterfire/blob/02a9be2645b9512b4ee0562e4898e4ebe5d40c6f/packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_firestore.dart#L79) is public.

Reply:
In that case I don't like the use of positional parameters here. 

It would probably be more readable if we used named parameters